python/lsst/pipe/tasks/assembleChi2Coadd.py

```diff
@@ -132,16 +135,16 @@ class AssembleChi2CoaddConfig(pipeBase.PipelineTaskConfig,
     )
 
 
-class AssembleChi2CoaddTask(pipeBase.Task):
-    """Assemble a chi^2 (Kaiser) coadd from a collection of multi-band coadds
+class AssembleChi2CoaddTask(pipeBase.PipelineTask):
+    """Assemble a chi^2 coadd from a collection of multi-band coadds
 
-    See Kaiser 2001 for more information.
+    See Salazay et al. 1999 and Kaiser 2001 for more information.
```

Comment:
Preferred way of including references is through its own section. https://developer.lsst.io/python/numpydoc.html#references